fedmsg_meta_fedora_infrastructure/taskotron.py

```diff
@@ -39,5 +46,45 @@ def link(self, msg, **config):
         if msg['topic'].endswith('taskotron.result.new'):
             return msg['msg']['result'].get('log_url', '')
 
+    def objects(self, msg, **config):
+        packages = self.packages(msg, **config)
+        return set([
+            '/'.join([
+                msg['msg']['task'].get('name', ''),
+                package,
+                msg['msg']['result'].get('outcome', '')
+            ])
+            for package in packages
+        ])
+    def packages(self, msg, **config):
+        type = msg['msg']['task']['type']
+        if type == 'koji_build':
+            nvr = msg['msg']['task']['item']
+            name, version, release = nvr.rsplit('-', 2)
```

Comment:
I'm a bit uneasy when doing NEVR operations by hand, that's why we use low level tools for it (`rpmUtils` from yum in the old days, `hawkey.split_nevra()` nowadays). It would break if we sent you ENVR string (yum style), but I think we don't do it, we tried to convert everything to NEVR (dnf style) or NVR (because some services don't process epoch correctly).